tasmota/tasmota_xdrv_driver/xdrv_85_esp32_ble_eq3_trv.ino

```diff
@@ -480,16 +480,18 @@ int EQ3ParseOp(BLE_ESP32::generic_sensor_t *op, bool success, int retries){
 
     ResponseAppend_P(PSTR(",\"hassmode\":"));
     do {
+      //HASS allowed modes [“auto”, “off”, “cool”, “heat”, “dry”, “fan_only”]
       //0201283B042A
-      // its in auto
+      // If its in auto or holiday, set to auto
       if ((stat & 3) == 0) { ResponseAppend_P(PSTR("\"auto\"")); break; }
-      // it's set to 'OFF'
+      // If its in manual and 4.5°C, set to off
       if (((stat & 3) == 1) && (status[5] == 9)) { ResponseAppend_P(PSTR("\"off\"")); break; }
-      // it's actively heating (valve open)
+      // If its in manual above 4.5°C and valve is open, set to heat
```

Comment:
On Discord there is a discussion ongoing to remove the valve position from the logic and always set manual above 4.5 to heat. Maybe a third opinion would be good.

Reply:
Think we are the only ones who care. :-D

Leave as is. Usually the valve will only be 0 when the TRV is in off mode anyway.

So I would say: ready for checking and merging :-)